stats/lib/platform-stats-fetchers.js

```diff
@@ -164,3 +164,40 @@ export const fetchParticipantsSummary = async (pgPool) => {
     participant_count: Number(rows[0].count)
   }
 }
+
+/**
+ * @param {Queryable} pgPool
+ * @param {import('./typings.js').DateRangeFilter} filter
+ */
+export const fetchDailyDesktopUsers = async (pgPool, filter) => {
+  const { rows } = await pgPool.query(`
+    SELECT 
+      day::TEXT, 
+      platform, 
+      SUM(user_count)::INT AS total
+    FROM daily_desktop_users
+    WHERE day >= $1 AND day <= $2
+    GROUP BY GROUPING SETS ((day, platform), (day))
+    ORDER BY day, platform NULLS LAST;`,
```

Comment:
do you know in which case platform is null?

Reply:
Platform is `null` in case it's sum of all users for given day (we're using `GROUPING SETS` to group by (day, platform) and by day).